src/app/api/actions/ajouterUneNoteDeContexteAction.test.ts

```diff
@@ -0,0 +1,87 @@
+
+import { revalidatePath } from 'next/cache'
+import { describe, it } from 'vitest'
+
+import { ajouterUneNoteDeContexteAction } from './ajouterUneNoteDeContexteAction'
+import { AjouterNoteDeContexteAGouvernance } from '@/use-cases/commands/AjouterNoteDeContexteAGouvernance'
+
+describe('ajouter une note de contexte', () => {
+  it('quand une note de contexte est ajoutée avec tous ses champs, alors cela renvoie un succès et le cache de la page appelante est purgé', async () => {
+    // GIVEN
+    vi.mock('next/cache', () => ({
+      revalidatePath: vi.fn(),
+    }))
```

Comment:
Je sais que c'est souvent mis sur internet mais c'est presque interdit cette version.
```suggestion
    vi.spyOn(nextCache, 'revalidatePath').mockImplementationOnce(vi.fn())
```
Ce qui est plus correct car c'est bien un spy.